### PR TITLE
Estimated pair counts refresh

### LINKED ISSUE #182: Ability to refresh estimated pair counts
The `EstimatedPairCounts` class should have a `refresh` method

### PR DESCRIPTION
**Pull request by juliamcclellan**
Fix #182

### CODE REVIEW

File: tamr_unify_client/models/project/mastering.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def estimate_pairs(self):
         """Returns pair estimate information for a mastering project
 
         :return: Pairs Estimate information.
-        :rtype: :class:`~tamr_unify_client.models.project.estimated_pair_counts`
+        :rtype: :class:`~tamr_unify_client.models.project.estimated_pair_counts.EstimatedPairCounts`
```
Comment (nbateshaus) [inline tamr_unify_client/models/project/mastering.py]: nice

File: tests/unit/test_pair_counts.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        p = MasteringProject(self.unify, self._project_json)
+        responses.add(
+            responses.GET, f"{self._url_base}/{self._api_path}", json=self._project_json
+        )
+        generated = p.estimate_pairs()
+
+        created = EstimatedPairCounts.from_json(
+            self.unify, self._estimate_json, self._api_path
+        )
+        self.assertEqual(print(generated), print(created))
```
Comment (nbateshaus) [inline tests/unit/test_pair_counts.py]: I'm pretty sure you want `repr()` here instead of `print()`

File: tests/unit/test_pair_counts.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        updated["status"]["state"] = "SUCCEEDED"
+        responses.add(responses.GET, f"{self._url_base}/operations/24", json=updated)
+
+        estimate = EstimatedPairCounts.from_json(
+            self.unify, self._estimate_json, self._api_path
+        )
+        generated = estimate.refresh()
+
+        created = Operation.from_json(self.unify, updated)
+        self.assertEqual(print(generated), print(created))
```
Comment (nbateshaus) [inline tests/unit/test_pair_counts.py]: `repr()` rather than `print()`

File: tests/unit/test_pair_counts.py (hunk 2 above)
Comment (juliamcclellan) [reply tests/unit/test_pair_counts.py]: true

**Comment by juliamcclellan** [comment]
Ignore what I just pushed, still working on it

**Pull request merged**